draft-ietf-core-oscore-edhoc.md

```diff
@@ -552,7 +553,9 @@ For all entries, the Change Controller is IETF, and the reference is \[RFC-XXXX]
 
 IANA is requested to create a new "EDHOC Authentication Credential Types" registry within the "Ephemeral Diffie-Hellman Over COSE (EDHOC)" registry group defined in {{I-D.ietf-lake-edhoc}}.
 
-The registry uses the "Expert Review" registration procedure {{RFC8126}}. Expert Review guidelines are provided in {{review}}.
+As registration procedure, the registry uses either "Standards Action with Expert Review", or "Specification Required" per {{Section 4.6 of RFC8126}}. Expert Review guidelines are provided in {{review}}.
```

Comment:
Is that a "registration procedure"?  I'd say registration policy.

Reply:
Fixed in https://github.com/core-wg/oscore-edhoc/commit/6d45a8c58598679157ee0ff6ced4cca0825e788c